common/app/views/fragments/email/signup/subscription/emailSignUp.scala.html

```diff
@@ -84,11 +86,13 @@ <h2 class="email-sub__heading">@Html(emailEmbedData.title)</h2>
         document.getElementById("@inputId")
         .addEventListener("focusin", ()=> {
             document.getElementById("@privacyNoticeId")?.setAttribute('class', '@privacyNoticeVisibleClass')
+            document.getElementById("@termsId")?.setAttribute('class', 'terms @termsDark')
```

Comment:
If you use `classList.add("my-class");` then you don't need to worry about overwriting any other existing classes

https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/add

Reply:
nice

Reply:
👍